packages/elements/src/chart/index.ts

```diff
@@ -447,24 +201,12 @@ export class Chart extends BasicElement {
    * @returns {void}
    */
   protected createChart (): void {
-    const ctx = this.canvas.getContext('2d');
-    if (ctx && this.config) {
-
-      // Are we reusing an old chart canvas?
-      const isReusingCanvas = this.destroyChart();
-
-      // Preparing the resources before create chart
+    if (this.canvas && this.config) {
```

Comment:
Let's check for canvas content as well:
```suggestion
    const ctx = this.canvas.getContext('2d');
      if (ctx && this.config) {
```